BTCPayServer/Controllers/GreenField/UsersController.cs

```diff
@@ -64,6 +64,39 @@ public class UsersController : ControllerBase
             _userService = userService;
         }
 
+        [Authorize(Policy = Policies.CanViewUsers, AuthenticationSchemes = AuthenticationSchemes.Greenfield)]
+        [HttpGet("~/api/v1/users/{idOrEmail}")]
+        public async Task<ActionResult<ApplicationUserData[]>> GetUser(string idOrEmail)
+        {
+            // Try ID
+            var user = await _userManager.FindByIdAsync(idOrEmail);
+            if (user == null)
+            {
+                // Try Email
+                user = await _userManager.FindByEmailAsync(idOrEmail);
+            }
+            if (user != null)
+            {
+                return Ok(await FromModel(user));
+            }
+
+            return NotFound();
```

Comment:
```suggestion
            return UserNotFound();
```
this will fix your test

Reply:
The return types don't match. How should I fix?
![image](https://user-images.githubusercontent.com/71019/146896368-814a756b-5d3d-4d41-8844-2f8fb8781607.png)

Reply:
Change the return type of the method to `Task<IActionResult>`

Reply:
Done.